ztictl/README.md

```
@@ -257,13 +271,13 @@ ztictl ssm exec-multi --all-regions --tags "Type=api" "status"
 
 For complete command documentation, see [docs/COMMANDS.md](../docs/COMMANDS.md).
 
-```
+````
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)